src/components/common/SearchBar/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import searchIcon from "assets/images/search.svg";
+
+const SearchBar = ({ placeholder }) => {
+  return (
+    <React.Fragment>
```

Comment:
And BTW there is a [shorthand](https://reactjs.org/docs/fragments.html#short-syntax) for this 
```suggestion
    <Fragment>
```